docs/source/how_to_guides/configure_cloud_storage_credentials.md

```diff
@@ -206,3 +207,29 @@ export AZURE_ACCOUNT_NAME='test'
 export AZURE_ACCOUNT_ACCESS_KEY='NN1KHxKKkj20ZO92EMiDQjx3wp2kZG4UUvfAGlgGWRn6sPRmGY/TEST/Dri+ExAmPlEExAmPlExA+ExAmPlExA=='
 ```
 ````
+
+## Databricks File System (DBFS)
+
+To upload data to and download data from the Databricks File System (DBFS), users must set their Databricks host (`DATABRICKS_HOST`) and access token (`DATABRICKS_TOKEN`) in the run environment.
+
+See the [Databricks documentation](https://docs.databricks.com/en/dev-tools/auth.html#databricks-personal-access-token-authentication) for instructions on how to create a personal access token.
+
+```{code-tab} py
+import os
+os.environ['DATABRICKS_HOST'] = 'hostname'
+os.environ['DATABRICKS_TOKEN'] = 'token key'
+```
+
+```{code-tab} sh
+export DATABRICKS_HOST='hostname'
+export DATABRICKS_TOKEN='token key'
+```
+
+### MosaicML platform
```

Comment:
Can you put this at the start of the section in `Databricks File System (DBFS)` to inline with other structure ?

Reply:
Done